unit-tests/live/safety/test-safety-preset-read-write-real-data.py

```diff
@@ -0,0 +1,98 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#test:donotrun
+#test:device D585S
+
+import pyrealsense2 as rs
+import random
+from rspy import test
+
+#############################################################################################
+# Tests
+#############################################################################################
+
+test.start("Verify Safety Sensor Extension")
+ctx = rs.context()
+dev = ctx.query_devices()[0]
+safety_sensor = dev.first_safety_sensor()
+test.finish()
+
+#############################################################################################
+
+test.start("New safety preset creation")
+
+# rotation matrix [-2..2] (no units)
+rx = rs.float3(random.uniform(-2, 2), random.uniform(-2, 2), random.uniform(-2, 2))
+ry = rs.float3(random.uniform(-2, 2), random.uniform(-2, 2), random.uniform(-2, 2))
+rz = rs.float3(random.uniform(-2, 2), random.uniform(-2, 2), random.uniform(-2, 2))
+rotation = rs.float3x3(rx, ry, rz)
+
+# translation vector [m]
+translation = rs.float3(random.uniform(0, 100), random.uniform(0, 100), random.uniform(0, 100))
+
+# init safety extrinsics table (transformation link) from rotation matrix and translation vector
+transformation_link = rs.safety_extrinsics_table(rotation, translation)
+
+# init safety platform config
+safety_platform = rs.safety_platform()
+safety_platform.transformation_link = transformation_link
+safety_platform.robot_height = random.uniform(1, 3)  # [m]
+safety_platform.robot_mass = random.uniform(20, 200)  # [kg]
+safety_platform.reserved = random.sample(range(0, 255), 16)
+
+# define 4 points to be used in polygon
+p1 = rs.float2(random.uniform(-10, 10), random.uniform(-10, 10))
+p2 = rs.float2(random.uniform(-10, 10), random.uniform(-10, 10))
+p3 = rs.float2(random.uniform(-10, 10), random.uniform(-10, 10))
+p4 = rs.float2(random.uniform(-10, 10), random.uniform(-10, 10))
+
+# zone boundary
+p5 = rs.float2(random.uniform(0, 10), random.uniform(0, 10))
```

Comment:
p1-p4 is understood, why use p5 & p6 for a non point elements?

Reply:
changed them to more clear variable names